src/app/EventManagement.cpp

```diff
@@ -318,26 +315,6 @@ CHIP_ERROR EventManagement::ConstructEvent(EventLoadOutContext * apContext, Even
     eventReportBuilder.Init(&(apContext->mWriter));
     // TODO: Update IsUrgent, issue 11386
     // TODO: Update statusIB, issue 11388
```

Comment:
Are these comments still valid?

Reply:
yes, we would figure out how to add statusIB after te7.5, currently we have not yet figured out how to achieve this. In this PR, we would disable event status IB processing.